wgsl/index.bs

```diff
@@ -1053,14 +1047,21 @@ case_body
 <pre class='def'>
 loop_stmt
   : LOOP BRACKET_LEFT statements continuing_stmt? BRACKET_RIGHT
-
-continuing_stmt:
-  : CONTINUING body_stmt
 </pre>
 
-The loop statement is one of the biggest divergences from other shader
-languages. The structure of the statement is designed to closely match the block
-ordering requirements for a loop in SPIR-V.
+The loop construct causes a block of statements, the *loop body*, to execute repeatedly.
+
+This repetition can be interrupted by a [[#break-statement]], `return`, or `kill`.
+
+Optionally, the last statement in the loop body may be a 
+[[#continuing-statement]].
+It is only required when the loop body has a [[#continue-statement]].
```

Comment:
Do we really need `continuing` if continue is present? A program could, e.g.:
```
if (x % 2 == 0) {
  i = i + 2;
  continue;
} else {
  i = i + 1;
  continue;
}
```

Reply:
Right. We could eliminate the requirement; the text for the continue statement would need to be updated.

Reply:
Done.

Reply:
I didn't put this in there, but translation of those "continue without a continuing" to SPIR-V would require that continue to be a forward branch, because the loop can only have one backedge block.

Reply:
Right, it would have to be equivalent to having an empty continuing-block.